lib/build/loader.js

```diff
@@ -76,8 +77,38 @@ exports.createRequireJSConfig = function createRequireJSConfig(platform, bundler
   return config;
 };
 
-exports.createSystemJSConfig = function createSystemJSConfig(platform) {
-  throw new Error('SystemJS is not yet supported');
+exports.createSystemJSConfig = function createSystemJSConfig(platform, bundler) {
+  let loaderOptions = bundler.loaderOptions;
+  let loaderConfig = bundler.loaderConfig;
+  let bundles = bundler.bundles;
+  let configName = loaderOptions.configTarget;
+  let includeBundles = shouldIncludeBundleMetadata(bundles, loaderOptions);
+  let bundleMetadata = {};
+  let config = {
+    map: {}
+  };
+  let location = platform.baseUrl || platform.output;
+  
+  for (let i = 0; i < bundles.length; ++i) {
```

Comment:
this section does work, but switching it to a functional method chaining -> bundles.map.filter.reduce might make this section more readable. Although this is definitely a nitpick ;)